### PR TITLE
Add qtkeychain 0.9.1 support to Linux/macOS build. 

### LINKED ISSUE #40: Mac/Linux: QtKeychain support
This is the final step to solve bug https://bugs.launchpad.net/mixxx/+bug/1642765
https://github.com/frankosterfeld/qtkeychain

### PR DESCRIPTION
**Pull request by daschuer**
This is the successor of https://github.com/mixxxdj/buildserver/pull/49 updated to qtkeychain 0.9.1

Fixes #40.

### CODE REVIEW

**Comment by daschuer** [comment]
Ok This finally works. I think we can merge this because it is also used for master builds.

File: scripts/macosx/build_qt5.sh [hunk 1]
```diff
@@ -29,8 +29,6 @@ export DISABLE_FFAST_MATH=yes
 source $PROGDIR/environment.sh
 export QTDIR=$MIXXX_PREFIX/Qt-${VERSION_NUMBER}
 
-echo "Building $VERSION for $MIXXX_ENVIRONMENT_NAME for architectures: ${QT_ARCHS[@]}"
```
Comment (rryan) [inline scripts/macosx/build_qt5.sh:32]: Why remove this line? It's in every build_ file so I'd prefer to keep it for consistency.

Comment (daschuer) [reply scripts/macosx/build_qt5.sh:32]: It was redundant. See line 18 above.

**Comment by daschuer** [comment]
Merge?

File: scripts/linux-ubuntu/pbuilderrc [hunk 2]
```diff
@@ -1,5 +1,5 @@
-QT4_PACKAGES="libqt4-dev libqt4-opengl libqt4-svg libqt4-xmlpatterns libqt4-opengl-dev libqt4-sql libqt4-sql-sqlite"
```
Comment (rryan) [inline scripts/linux-ubuntu/pbuilderrc:1]: one more place is needed:
https://github.com/mixxxdj/buildserver/blob/dc77e009332ad9f002b223841e2090f9f92f2742/fabfile.py#L24

File: scripts/macosx/build_qt5.sh (hunk 1 above)
Comment (rryan) [reply scripts/macosx/build_qt5.sh:32]: Oops, nevermind!

File: scripts/macosx/build_qtkeychain.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# Quit on errors
+set -e
+
+# Get the path to our scripts folder.
+pushd `dirname $0` > /dev/null
+PROGDIR=`pwd -P`
+popd > /dev/null
+
+export VERSION_NUMBER=0.9.1
+export QT_VERSION=5.10.1
```
Comment (rryan) [inline scripts/macosx/build_qtkeychain.sh]: this is now 5.11.1

**Comment by daschuer** [comment]
`No space left on device`

Can one have a look?

**Comment by Be-ing** [comment]
test this please

**Comment by Be-ing** [comment]
@Pegasus-RPG cleared up disk space a few days ago, so I think it should build now.

File: scripts/macosx/build_environment.sh [hunk 4]
```diff
@@ -16,6 +16,7 @@ MIXXX_MACOSX_SDK=$(xcodebuild -version -sdk macosx SDKVersion)
 # Qt 5.11 requires a minimum of macOS 10.11.
 MIXXX_MACOSX_TARGET='10.11'
 MIXXX_MACOSX_STDLIB='libc++'
+MIXXX_QT_VERSION='5.10.1'
```
Comment (Be-ing) [inline scripts/macosx/build_environment.sh]: Please rebase this to use Qt 5.11.1

**Comment by daschuer** [comment]
Ready:-)

**Comment by rryan** [comment]
thanks!

**Pull request merged**